src/main/java/org/sonarlint/intellij/finding/LiveFinding.java

```diff
@@ -203,4 +208,10 @@ public Optional<FindingContext> context() {
   public List<QuickFix> quickFixes() {
     return quickFixes;
   }
+
+  @org.jetbrains.annotations.Nullable
```

Comment:
Don't we use javax annotations usually ?

Reply:
This is not very important I guess, since annotations are not loaded at runtime. I did this change to silence an InteliiJ inspection, because of the Kotlin -> Java conversion. As we will continue to migrate the codebase to Kotlin, this will disappear.